index.bs

```diff
@@ -1619,7 +1693,114 @@ QUIC streams.
 Stats {#streaming-stats}
 ------------------------------
 
-TODO
+Durings a streaming session, the sender should send stats with the
+streaming-session-sender-stats-event at the interval the receiver
+requested.  The streaming-session-sender-stats-event message contains
+the following properties:
+
+: streaming-session-id
+:: The ID of the streaming session these stats apply to.
+
+: system-time
+:: The time when the stats were calculated, using a monotonic system
+    clock.
+
+: audio
+:: Stats specific to audio.  Stats for multiple encodings can be sent
+    at once, but encodings need not be included if the stats haven't
+    changed.  See below.
+
+: video
+:: Stats specific to video.  Stats for multiple encodings can be sent
+    at once, but encodings need not be included if the stats haven't
+    changed.  See below.
+
+Audio encoding sender stats include the following properties:
+
+: encoding-id
+:: The ID of the encoding for which the stats apply.
+
+: cumulative-sent-frames
```

Comment:
Is there an advantage to cumulative vs. incremental metrics for these stats?

Reply:
Yes, I mentioned them in the previous comment.  It's how WebRTC does stats as well.  It's a good technique, I think.

Reply:
SGTM